package.json

```diff
@@ -74,6 +74,7 @@
     "@babel/core": "^7.7.5",
     "@babel/preset-typescript": "^7.7.4",
     "@types/cryptr": "^4.0.0",
+    "@types/file-saver": "^2.0.1",
```

Comment:
@SferaDev  Why is file saver now a dev dependency? Did you do this on purpose?

Reply:
@SferaDev I am merging this because everything seems to be working fine and I want to finish reviewing the pending PRs but please let me know about the file saver dependency. We can always fix it in another PR if needed. Thanks!

Reply:
This is the type definitions. Previously we only used the file saver lib on JS files but with this refactor we moved the code to a TS one. The type definitions are now needed and we include them always as dev dependencies.